bin/plotting/pycbc_plot_vt_ratio

```diff
@@ -0,0 +1,103 @@
+parser.add_argument('--comp-2-file', required=True,
+                    help='HDF file containing VT curves, second set of data for comparison')
+parser.add_argument('--comp-1-desc',  required=True,
+                    help='descriptor tag for first set of data (short, for use in subscript)')
+parser.add_argument('--comp-2-desc', type=str, required=True,
+                    help='descriptor tag for second set of data (short, for use in subscript)')
+parser.add_argument('--outfile', type=str, required=True,
+                    help='Output file to save to')
+parser.add_argument('--ifars', type=float, required=True, nargs='+',
+                    help='IFAR values to plot VT for')
+
+args = parser.parse_args()
+
+# Load in the two datasets
+ffirst_set = h5py.File(args.comp_1_file)
+fsecond_set = h5py.File(args.comp_2_file)
+
+# Find the index closest to the given IFAR value
+idxs = [np.argmin(np.abs(ffirst_set['xvals'][:] - ifv)) for ifv in args.ifars]
+
+keys = ffirst_set['data'].keys()
+# sanitise the input a little bit so that the files must have the same 
+assert keys == fsecond_set['data'].keys(), "keys do not match for the two files"
+
+# make the plot pretty
+plt.rc('axes.formatter', limits=[-3,4])
+plt.rc('figure', dpi=300)
+fig_mi = plt.figure(figsize=(10,4))
+ax_mi = fig_mi.gca()
+
+ax_mi.grid(True, zorder=1) #add a grid, set zorder to be low so that it is behind the rest
+
+# read in labels for the different plotting points
+labels = ['$ ' + label.split('\\in')[-1] for label in ffirst_set['data'].keys()]
+
+# read in the splitting parameter name from the first data set
+x_param = r'$' + ffirst_set['data'].keys()[0].split('\\in')[0].strip('$').strip() + r'$'
+
+# read in the positions from the labels
+xpos = np.array([float(l.split('[')[1].split(',')[0]) for l in labels])
+# TODO?: add in argument for equally spaced plotting (e.g. for split-in-log parameters)
+
+# set the x ticks to be the positions given in the labels
+plt.xticks(xpos,labels,rotation='horizontal')
+
+#loop through each IFAR and plot the VT ratio with error bars
+for idv in idxs:
+	data = np.array([ffirst_set['data'][key][idv] for key in keys])
+	data_2 = np.array([fsecond_set['data'][key][idv] for key in keys])
+	
+	sets_errhigh = np.array([ffirst_set['errorhigh'][key][idv] for key in keys])
+	sets_errlow = np.array([ffirst_set['errorlow'][key][idv] for key in keys])
+	
+        sets_2_errhigh = np.array([fsecond_set['errorhigh'][key][idv] for key in keys])
+        sets_2_errlow = np.array([fsecond_set['errorlow'][key][idv] for key in keys])
+
+	ys = np.divide(data, data_2)
+	yerr_errlow = np.multiply(np.sqrt(np.divide(sets_errlow,data)**2 +
+                np.divide(sets_2_errlow,data_2)**2),ys)
+	yerr_errhigh = np.multiply(np.sqrt(np.divide(sets_errhigh,data)**2 +
+                np.divide(sets_2_errhigh,data_2)**2),ys)
+	ax_mi.errorbar(xpos, ys, yerr=[yerr_errlow,yerr_errhigh],
+                fmt='o', markersize=7, alpha=0.6, linewidth=5,
```

Comment:
I'll leave this to your discretion, but I typically like to specify colors for each trace to avoid having matplotlib's defaults change out from under me. Something like:
```
colors = ['blue', 'orange']
for idv, c in zip(idxs, colors):
    do loop things and plot
```

Reply:
sounds good - I'm quite keen in general on using colorblind-friendly schemes, so have put in one that has been suggested [here](https://github.com/matplotlib/matplotlib/issues/9460#issuecomment-377312565).

Because of the update to allow a generic number of ifar values, I've not used the zipping suggestion, rather using a counter which loops back to the start of the colors list when it gets to that point